Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.8-slim-buster
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /civi_app
+
+WORKDIR /civi_app
+
+ADD . /civi_app
+
+RUN apt-get update && apt-get install -y git

```

Comment:
Which of our dependencies requires git?

Reply:
The `django-notifications` module is being installed using git. 

<img width="428" alt="image" src="https://user-images.githubusercontent.com/38958532/175227928-8ded7dfb-492f-4166-86fb-2471232d1178.png">

Reply:
OK, lets see if we can change that dependency to use a regular pip package rather than git, so our development dependencies are simpler.

Reply:
I was going through the commit history and I think @gorkemarslan changed it to the git version. Maybe we should start by discussing why this was done in the first place.